libraries/src/Helper/PublicFolderGeneratorHelper.php

```diff
@@ -161,19 +161,18 @@ public function createPublicFolder(string $destinationPath): void
         // The API root index.php
         $this->createFile($fullDestinationPath . 'api/index.php', str_replace(['{{APPLICATIONPATH}}', '{{DEFINESPATH}}'], ['\'' . DIRECTORY_SEPARATOR . 'api\'', 'dirname(__DIR__)'], $this->indexTemplate));
 
-        if (is_dir(JPATH_ROOT . '/images')) {
-            $this->createSymlink($root . 'images', $destinationPath . 'images', JPATH_ROOT . '/');
-        }
-
-        // Create symlinks for all the local filesystem directories
-        if (PluginHelper::isEnabled('filesystem', 'local')) {
+        // Get all the local filesystem directories
+        if (defined('_JCLI_INSTALLATION')) {
```

Comment:
```suggestion
        if (\defined('_JCLI_INSTALLATION')) {
```
Fix PHPCS failing in drone.

Reply:
Did it already but thanks